specification/maps/data-plane/Search/preview/2023-04-01-preview/search.json

```diff
@@ -0,0 +1,1437 @@
+      "required": false,
+      "type": "array",
+      "items": {
+        "type": "string",
+        "enum": [
+          "Address",
+          "Neighborhood",
+          "PopulatedPlace",
+          "Postcode1",
+          "AdminDivision1",
+          "AdminDivision2",
+          "CountryRegion"
+        ],
+        "x-ms-enum": {
+          "name": "ReverseGeocodingResultTypeEnum",
+          "modelAsString": true,
+          "values": [
+            {
+              "name": "Address",
+              "value": "Address"
+            },
+            {
+              "name": "Neighborhood",
+              "value": "Neighborhood"
+            },
+            {
+              "name": "PopulatedPlace",
+              "value": "PopulatedPlace"
+            },
+            {
+              "name": "Postcode1",
+              "value": "Postcode1"
+            },
+            {
+              "name": "AdminDivision1",
+              "value": "AdminDivision1"
+            },
+            {
+              "name": "AdminDivision2",
+              "value": "AdminDivision2"
+            },
+            {
+              "name": "CountryRegion",
+              "value": "CountryRegion"
+            }
+          ]
+        }
+      },
+      "x-ms-parameter-location": "method"
+    }
+  },
+  "paths": {
+    "/geocode": {
+      "get": {
+        "description": "**Geocoding**\n\n**Applies to:** see pricing [tiers](https://aka.ms/AzureMapsPricingTier).\n\nIn many cases, the complete search service might be too much, for instance if you are only interested in traditional geocoding. Search can also be accessed for address look up exclusively. The geocoding is performed by hitting the geocoding endpoint with just the address or partial address in question. The geocoding search index will be queried for everything above the street level data. No Point of Interest (POIs) will be returned. Note that the geocoder is very tolerant of typos and incomplete addresses. It will also handle everything from exact street addresses or street or intersections as well as higher level geographies such as city centers, counties, states etc.\n>[!Important]\n>By using this feature, you agree to the preview legal terms. See the [Preview Supplemental Terms](https://azure.microsoft.com/support/legal/preview-supplemental-terms/) for additional details.",
```

Comment:
Remove "\n\n**Applies to:** see pricing [tiers](https://aka.ms/AzureMapsPricingTier)."